workloads/storage/pvcscale_README.md

```diff
@@ -0,0 +1,116 @@
+### Purpose
+PVC scale test is designed to create desired number of pods and attach PVC to every pod from storage backend dynamically 
+using storage storage class. 
+
+### Prerequesties
+- PVC test requieres functional storage backend and storage class. 
+- Pbench [web server](https://distributed-system-analysis.github.io/pbench/doc/server/pbench-web-server.html)
+
+### Environment variables for PVC Scale workload playbook
+
+```
+###############################################################################
+# Ansible SSH variables.
+###############################################################################
+export PUBLIC_KEY
+export PRIVATE_KEY
+
+ORCHESTRATION_USER="root"
+###############################################################################
+# RHCOS workload variables.
+###############################################################################
+export WORKLOAD_IMAGE="quay.io/openshift-scale/scale-ci-workload"
+export KUBECONFIG_FILE="path to kube config file" 
+export PBENCH_SSH_PRIVATE_KEY_FILE
+export PBENCH_SSH_PUBLIC_KEY_FILE
+export ENABLE_PBENCH_AGENTS=true
+export PBENCH_SERVER="fqdn of pbench web server" 
+
+export SCALE_CI_RESULTS_TOKEN
+export JOB_COMPLETION_POLL_ATTEMPTS
+
+# PVC Scale workload specific parameters:
+export PVCSCALE_TEST_PREFIX="test prefix"
```

Comment:
Great you're trying to document the vars.  I'd keep the docs for the vars consistent across the workloads though.  Granted, there's not much of a documentation for these, but shouldn't we use standard comments itself, like `# comment`?

Reply:
ok, will change and we will have to document for other tests parameters too

Reply:
I will drop README from this PR and create new one on top of master per #16